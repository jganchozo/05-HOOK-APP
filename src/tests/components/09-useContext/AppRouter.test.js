import { mount } from 'enzyme';
import React from 'react';
import { AppRouter } from '../../../components/09-useContext/AppRouter';
import { UserContext } from '../../../components/09-useContext/UserContext';

describe('Pruebas en AppRouter', () => {

    const user = {
        name: 'Jose',
        email: 'jganchozo@outlook.com'
    }

    const wrapper = mount(
        <UserContext.Provider value={{
            user
        }}>
            <AppRouter />
        </UserContext.Provider>
    );

    test('should de omostrarse correctamente', () => {
     expect(wrapper).toMatchSnapshot();   
    });

});