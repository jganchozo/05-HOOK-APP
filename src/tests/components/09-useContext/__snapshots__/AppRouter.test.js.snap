// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en AppRouter should de omostrarse correctamente 1`] = `
<div>
  <nav
    className="navbar navbar-expand-sm navbar-dark bg-dark"
  >
    <a
      className="navbar-brand"
      href="/"
      onClick={[Function]}
    >
      useContext
    </a>
    <div
      className="collapse navbar-collapse"
      id="navbarNavAltMarkup"
    >
      <div
        className="navbar-nav"
      >
        <a
          aria-current="page"
          className="nav-link navData => navData.isActive && 'active' active"
          href="/"
          onClick={[Function]}
        >
          Home
        </a>
        <a
          className="nav-link isActive => isActive && 'active'"
          href="/about"
          onClick={[Function]}
        >
          About
        </a>
        <a
          className="nav-link isActive => isActive && 'active'"
          href="/login"
          onClick={[Function]}
        >
          Login
        </a>
      </div>
    </div>
  </nav>
  <div
    className="container"
  >
    <div>
      <h1>
        HomeScreen
      </h1>
      <hr />
      <pre>
        {
   "name": "Jose",
   "email": "jganchozo@outlook.com"
}
      </pre>
    </div>
  </div>
</div>
`;
